Examples/UtiliKit-iOSExample/Collection/Cells, Supplementaries and Decorations/Cells/ProgrammaticCell.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  ProgrammaticCell.swift
+//  UtiliKit
+//
+//  Copyright © 2017 Bottle Rocket Studios. All rights reserved.
+//
+
+import UIKit
+import UtiliKit
+
+class ProgrammaticCell: UICollectionViewCell { }
+
+// MARK: - Configurable
+extension ProgrammaticCell: Configurable {
+
+    func configure(with color: UIColor) {
```

Comment:
Similar comment here about exposing the notion that the background color is being configured.